waspc/src/Generator/Job.hs

```diff
@@ -0,0 +1,27 @@
+module Generator.Job
```

Comment:
Looking at Job now, I am realizing I tried to make it more general and not tied just to the processes, but on the other I say below it always returns ExitCode, and only types of output are String or exit code hm. To make it really general, it should probably be parametrized in 1. return type and 2. message data. But ok, maybe it is best to leave it for now?

Reply:
yes, let's leave it for now, I think it is obvious enough what is it about.